nuxeo-ai-internal/src/main/java/org/nuxeo/ai/internal/InitAudit.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * (C) Copyright 2018-2020 Nuxeo SA (http://nuxeo.com/).
+ *   This is unpublished proprietary source code of Nuxeo SA. All rights reserved.
+ *   Notice of copyright on this source code does not indicate publication.
+ *
+ *   Contributors:
+ *       Nuxeo
+ */
+
+package org.nuxeo.ai.internal;
+
+import static org.nuxeo.ai.internal.InitAudit.ID;
+
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
+
+import javax.ws.rs.core.Response;
+
+import org.nuxeo.ecm.automation.core.annotations.Context;
+import org.nuxeo.ecm.automation.core.annotations.Operation;
+import org.nuxeo.ecm.automation.core.annotations.OperationMethod;
+import org.nuxeo.ecm.automation.core.annotations.Param;
+import org.nuxeo.ecm.core.api.CoreInstance;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.api.NuxeoPrincipal;
+import org.nuxeo.ecm.platform.audit.api.AuditLogger;
+import org.nuxeo.ecm.platform.audit.api.ExtendedInfo;
+import org.nuxeo.ecm.platform.audit.api.LogEntry;
+import org.nuxeo.ecm.platform.audit.impl.ExtendedInfoImpl;
+import org.nuxeo.runtime.api.Framework;
+
+@Operation(id = ID, category = "AICore", label = "Init Audit", description = "Init audit for metrics")
+public class InitAudit {
```

Comment:
IF the sole purpose of this operation is for tests, should it not be part of the test code ?

Reply:
Yes it could but as we have in the nuxeo ai cloud repository, an internal module could be useful to integrate for migrating elements as well. We talked about it quickly with julien and it was not shocking to have a dedicated module for those kind of things

Reply:
Actually I update my comment: this operation wont be used for unit testing - only for giving examples to dev on the front client side.